includes/class-sensei-lesson.php

```diff
@@ -287,8 +290,23 @@ public function lesson_prerequisite_meta_box_content() {
 			'suppress_filters' => 0,
 			'post_status'      => [ 'publish', 'draft' ],
 		);
+
+		if ( $current_lesson_course ) {
+			// Meta query args to only get Lesson Posts from current Course.
+			$meta_query_args = array(
+				array(
+					'key'     => '_lesson_course',
+					'value'   => $current_lesson_course,
+					'compare' => '=',
+				),
+			);
+				// Add meta query to $post_args.
```

Comment:
There's an extra level of indentation here. I'm surprised PHPCS didn't pick up on it.

Reply:
Done!